cli/internal/run/run_state.go

```diff
@@ -77,12 +62,11 @@ type RunState struct {
 	Attempted int
 
 	startedAt time.Time
-	config    *config.Config
 }
 
 // NewRunState creates a RunState instance for tracking events during the
 // course of a run.
-func NewRunState(startedAt time.Time, tracingProfile string, config *config.Config) *RunState {
```

Comment:
I'm trying real hard to remember why I added this - but I got nothing. It must have been for part of the cache permissions work that got pulled out during the review process and this was left behind. In any event, thanks for cleaning it up!